### ISSUE TITLE
Anti-myeloperoxidase (MPO) antibody positivity

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6597]
**Preferred term label:** Anti-myeloperoxidase (MPO) antibody positivity

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Perinuclear antineutrophil antibody positivity HP:0032229

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
